### PR TITLE
Correct code after renaming profile - api/profiles

### PR DESCRIPTION
**Pull request by andsemenov**
Now it doesn't create applicant's profiles.

### CODE REVIEW

**Review by DwinaTech — DISMISSED** [review]
👍🏿

**Comment by andsemenov** [comment]
> 👍🏿

I added some changes which correct double records during a creation a new profile, changed await for experiense in server/api/profiles. Could you review it again, please? Thank you.

File: server/api/profiles.js [hunk 1]
```diff
@@ -1,40 +1,50 @@
...
+  passport.authenticate("jwt", { session: false }),
+  async (req, res) => {
+    try {
+      const db_newProfile = req.body;
+      db_newProfile.applicant_id = req.user.id;
+      db_newProfile.profile_public_id = nanoid();
+      const { experiences } = req.body;
+      const newProfile = await createProfile(db_newProfile);
+      experiences.forEach(async (experience) => {
+        await createWorkExperience({
```
Comment (gregdyke) [inline server/api/profiles.js:25]: As mentioned on slack:

```suggestion
        await Promise.all(experiences.map((experience) => createWorkExperience({ ...experience, profile_id: response[0].id }));
```

File: server/api/profiles.js [hunk 2]
```diff
@@ -1,40 +1,50 @@
...
+  "/",
+  passport.authenticate("jwt", { session: false }),
+  async (req, res) => {
+    try {
+      const db_newProfile = req.body;
+      db_newProfile.applicant_id = req.user.id;
+      db_newProfile.profile_public_id = nanoid();
+      const { experiences } = req.body;
+      const newProfile = await createProfile(db_newProfile);
+      experiences.forEach(async (experience) => {
```
Comment (gregdyke) [inline server/api/profiles.js]: as mentioned on slack:

await Promise.all(experiences.map((experience) => createWorkExperience({ ...experience, profile_id: newProfile.id })));

**Review by gregdyke — CHANGES_REQUESTED** [review]
sorry, I gave some incorrect advice on the await experiences

**Review by gregdyke — APPROVED** [review]
Looks good to me. We should still figure out why Andrei's vscode is doing so much formatting that everyone else's isn't :(

**Pull request merged**